### PR TITLE
Correct doc url added to README

### PR DESCRIPTION
**Pull request by rishatiwari**
### Description (*)

Correct doc url added to README

### Fixed Issues (if relevant)

None

### Manual testing scenarios (*)

Check `phpserver/README.md`

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: phpserver/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ php bin/magento setup:install --base-url=http://127.0.0.1:8082
 Notes:
 - By default, Magento creates a random Admin URI for you. Make sure to write this value down because it's how you access the Magento Admin later. For example : ```http://127.0.0.1:8082/index.php/admin_1vpn01```.
 
-For more informations about the installation process using the CLI, you can consult the dedicated documentation that can found in [the developer documentation](https://github.com/magento/devdocs/blob/develop/guides/v2.0/install-gde/install/cli/install-cli-install.md).
+For more information about the installation process using the CLI, you can consult the dedicated documentation that can found in [the developer documentation](https://github.com/magento/devdocs/blob/master/src/guides/v2.3/install-gde/install/cli/install-cli-install.md).
```
Comment (lbajsarowicz) [inline phpserver/README.md]: https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands.html
Why don't you link to Documentation itself, just to documentation source?

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Please read comment attached.

File: phpserver/README.md (hunk 1 above)
Comment (rishatiwari) [reply phpserver/README.md]: thanks. done

**Review by lbajsarowicz — APPROVED** [review]
Thanks for your contribution.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6789](https://jira.corp.magento.com/browse/ENGCOM-6789) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@rishatiwari thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

**Review by ghost — CHANGES_REQUESTED** [review]
magento:2.4-develop

**Pull request merged**